lib/screens/organisation_registration/organisation_registration.dart

```diff
@@ -0,0 +1,87 @@
+import 'dart:io';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'widgets/name_field.dart';
+import 'widgets/tagline_field.dart';
+import 'package:image_picker/image_picker.dart';
+class AddOrganisation extends StatefulWidget {
```

Comment:
1. Make sure the name of a widget ends with what it actually is.
```suggestion
class AddOrganisationScreen extends StatefulWidget {
```
2. Make sure the name of the file and name of widget in it is related
```suggestion
class OrganisationRegistrationScreen extends StatefulWidget {
```